.github/workflows/test-coverage.yml

```diff
@@ -0,0 +1,169 @@
+name: Test Coverage Report
+
+on:
+  push:
```

Comment:
Should this have been `pull_request`, assuming it's only there for testing purposes?

Reply:
For now, the idea is just to surface the report so anyone interested in writing tests can find an area to focus on.

We could submit the reports to a third party service. Do you have any that you recommend?

Reply:
`push` is just for testing on my fork. I hadn't thought about generating on pull request. That could be nice eventually.